src/main/views/cookies.njk

```diff
@@ -0,0 +1,91 @@
+{% extends "layout.njk" %}
+{% set heading = t('Cookies') %}
+
+{% set title = t('Money claim') %}
```

Comment:
translating isn't needed here it will be translated in the template, same for heading above